src/Surfnet/StepupSelfService/SelfServiceBundle/Service/ActivationFlowService.php

```diff
@@ -83,6 +83,11 @@ public function getPreference(): ActivationFlowPreferenceInterface
             ];
         }
 
+        if ($requestedActivationPreference instanceof ActivationFlowPreferenceNotExpressed && count($availableActivationPreferences) == 1) {
```

Comment:
Does PHPStan not complain about the unnecessarily vague expression here? 
```suggestion
        if ($requestedActivationPreference instanceof ActivationFlowPreferenceNotExpressed && count($availableActivationPreferences) === 1) {
```